webpack.config.js

```diff
@@ -6,12 +6,20 @@ const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries");
 const RemovePlugin = require('remove-files-webpack-plugin');
 const dashDash = require('@greenpeace/dashdash');
 
+const entryPoints = blockName => {
+  return {
+    [`${blockName}EditorScript`]: `./assets/src/blocks/${blockName}/${blockName}EditorScript.js`,
```

Comment:
I made these names match the WP standard: `editor-script`, `editor-style`, `script`, and `style`.  But CamelCased.